examples/window-app/nrfconnect/main/WindowCovering.cpp

```diff
@@ -35,10 +35,21 @@ using namespace chip::app::Clusters::WindowCovering;
 
 static const struct pwm_dt_spec sLiftPwmDevice = PWM_DT_SPEC_GET(DT_ALIAS(pwm_led1));
 static const struct pwm_dt_spec sTiltPwmDevice = PWM_DT_SPEC_GET(DT_ALIAS(pwm_led2));
-static k_timer sLiftTimer;
-static k_timer sTiltTimer;
+
 static constexpr uint32_t sMoveTimeoutMs{ 200 };
 
+static constexpr uint32_t FromOneRangeToAnother(uint32_t aInMin, uint32_t aInMax, uint32_t aOutMin, uint32_t aOutMax,
+                                                uint32_t aInput)
+{
+    const auto diffInput  = aInMax - aInMin;
+    const auto diffOutput = aOutMax - aOutMin;
+    if (diffInput > 0)
+    {
+        return static_cast<uint32_t>(aOutMin + static_cast<uint64_t>(aInput - aInMin) * diffOutput / diffInput);
```

Comment:
I'd order this such that the multiplication happens first before the division (to prevent loss of precision).

```suggestion
        return static_cast<uint32_t>(aOutMin + (static_cast<uint64_t>(aInput - aInMin) * diffOutput) / diffInput);
```

Reply:
This function has been deleted.